packages/cascara/src/ui/Columns/Columns.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import pt from 'prop-types';
+import { Role } from 'reakit';
+import classNames from 'classnames/bind';
+
+import Boundaries from '../../system-components/Boundaries';
+import styles from './Columns.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /** HTML tag to allow polymorphism */
+  as: pt.string,
+  /** The section content */
+  children: pt.oneOfType([pt.arrayOf(pt.node), pt.node]),
+  /** Columns can have css class name */
+  className: pt.string,
+  /** The number of columns to allocate */
+  count: pt.number,
```

Comment:
I think I missed this on the last review. Count should be either a number or the string 'auto'